port_drayage_plugin/src/port_drayage_worker.cpp

```diff
@@ -108,6 +128,7 @@ namespace port_drayage_plugin
                 _previous_strategy_params = msg->strategy_params;
                 
                 // Process event based on the PortDrayageEvent associated with the received MobilityOperation message
+                // TODO: Update this object's '_has_cargo' flag and '_cargo_id' based on the received status message from a loading/unloading device
```

Comment:
This todo is being covered by a separate story, correct?

Reply:
Correct, this is being addressed in CF-9 (CMV Broadcasts Arrival at Loading Point).